Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -46,6 +47,76 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
             return contactPoliciesSettings;
         }
 
+        public async Task<int?> GetUserIdAsync(string accountName)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+            const string query =
+                @"select IdUser from [User] u where u.Email = @Email;";
+            var queryParams = new { Email = accountName };
+            return await connection.QueryFirstOrDefaultAsync<int?>(query, queryParams);
+        }
+
+        public async Task UpdateContactPoliciesSettingsAsync(Entities.ContactPoliciesSettings contactPoliciesSettings)
+        {
+            using var connection = await _databaseConnectionFactory.GetConnection();
+
+            const string updateQuery =
+                @"UPDATE [UserShippingLimit] SET Active = @Active, Interval = @IntervalInDays, Amount = @EmailsAmountByInterval WHERE IdUser = @IdUser;";
```

Comment:
Just to continue with `GetUserIdAsync` method conventions.
```suggestion
                @"update [UserShippingLimit] set Active = @Active, Interval = @IntervalInDays, Amount = @EmailsAmountByInterval where IdUser = @IdUser;";
```

Reply:
ok i will be following the same convention for incoming implementations